code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -82,21 +86,32 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food)
 	//slicing food can be done here using sliceable == TRUE, slice_amount, and slice_product
 	//there will probably be a good deal of food that can be sliced but use their own slicing mechanisms instead of this, just gonna make a few things sliceable for now (aka just tomatoes, pepperoni, and cheese)
 	//might be a good idea to call a proc in the src obj and carry over the new src.slice_product so the src can fuck with the new obj as necessary?
+	//A proc was added :)
 	attackby(obj/item/W, mob/user)
 		if (src.sliceable && istool(W, TOOL_CUTTING | TOOL_SAWING))
 			var/turf/T = get_turf(src)
-			user.visible_message("[user] cuts [src] into [src.slice_amount] slices.", "You cut [src] into [src.slice_amount] slices.")
+			user.visible_message("[user] cuts [src] into [src.slice_amount] [src.slice_suffix]s.", "You cut [src] into [src.slice_amount] [src.slice_suffix]s.")
```

Comment:
`s_es()` ?

Reply:
I need a bit of clarrification. Is that a question about the "s" after [src.slice_suffix] or is that a question of i should add a variable for the plural of the slice?

Reply:
It's the name of a proc you want for this

Reply:
Oh, this is usefull to know, thanks!